simulator/src/main/java/com/hazelcast/simulator/common/SimulatorProperties.java

```
@@ -16,82 +16,61 @@
      *
      * @param file the file to load the properties from. If the file is null, then first the simulator.properties
      *             in the working dir is checked.
      */
     public void init(File file) {
         if (file == null) {
-            //if no file is explicitly given, we look in the working directory
-            File fallbackPropsFile = new File("simulator.properties");
-            if (fallbackPropsFile.exists()) {
-                file = fallbackPropsFile;
-            } else {
-                LOGGER.warn(format("%s is not found, relying on defaults", fallbackPropsFile));
+            // if no file is explicitly given, we look in the working directory
+            file = new File(PROPERTIES_FILE_NAME);
+            if (!file.exists()) {
+                LOGGER.warn(format("%s is not found, relying on defaults", file.getAbsolutePath()));
+                return;
             }
         }
 
-        if (file != null) {
-            LOGGER.info(format("Loading simulator.properties: %s", file.getAbsolutePath()));
-            load(file);
-        }
+        LOGGER.info(format("Loading %s: %s", PROPERTIES_FILE_NAME, file.getAbsolutePath()));
+        check(file);
+        load(file);
     }
 
-    private void load(File file) {
+    private void check(File file) {
         if (!file.exists()) {
-            exitWithError(LOGGER, "Could not find simulator.properties file: " + file.getAbsolutePath());
-            return;
+            exitWithError(LOGGER, format("Could not find %s file: %s", PROPERTIES_FILE_NAME, file.getAbsolutePath()));
         }
+    }
 
+    void load(File file) {
         FileInputStream inputStream = null;
         try {
             inputStream = new FileInputStream(file);
-            try {
-                properties.load(inputStream);
-            } catch (IOException e) {
-                closeQuietly(inputStream);
-            }
+            properties.load(inputStream);
```

This call should just fail if the `new FileInputStream(file)` would also fail. I couldn't find any file content which caused `properties.load()` to fail, so the ignored exception was not triggerable for me. So if something fails here I think we should see the exception.